src/utils/randomUtils.ts

```diff
@@ -1,7 +1,7 @@
 import * as crypto from 'crypto'
 
 export namespace randomUtils {
-  export function getPseudononymousStringHash(
+  export function getPseudoAnonymousStringHash(
```

Comment:
I think the correct term is `Pseudonymous` (I haven't heard about this term before)

Reply:
my bad 😅
never heard `Pseudononymous` term

Reply:
fixed